micro-ui/web/micro-ui-internals/packages/modules/pgr/src/configs/UICustomizations.js

```
@@ -0,0 +1,281 @@
+  "muster-roll-approval": "muster-roll-service",
+};
+
+export const UICustomizations = {
+  businessServiceMap,
+  updatePayload: (applicationDetails, data, action, businessService) => {
+    if (businessService === businessServiceMap.estimate) {
+      const workflow = {
+        comment: data.comments,
+        documents: data?.documents?.map((document) => {
+          return {
+            documentType: action?.action + " DOC",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)